crates/uv-dev/src/compile.rs

```diff
@@ -26,6 +26,8 @@ pub(crate) async fn compile(args: CompileArgs) -> anyhow::Result<()> {
             &PythonRequest::default(),
             EnvironmentPreference::OnlyVirtual,
             &cache,
+            // FIXME: Parse this from CLI
```

Comment:
It looks like in `uv-dev` we don't currently parse the preview flag. At the moment I am passing `PreviewMode::Disabled` directly here. How do we want to handle the preview flag in this context?

Reply:
I don't think we really need it. I don't use the `uv-dev` commands anymore, personally.